packages/config/src/layer2s/zksyncera.ts

```diff
@@ -61,6 +65,11 @@ export const zksyncera: Layer2 = {
         tokens: ['USDC', 'PERP', 'MUTE'],
       },
     ],
+    transactionApi: {
+      type: 'rpc',
+      startBlock: 1,
+      excludeFromActivityApi: true, // excluding until it's synced
+    },
```

Comment:
You need to include the url here, rn it will only work locally (it's in `config.local.ts`)
```suggestion
    transactionApi: {
      type: 'rpc',
      startBlock: 1,
      url: 'https://mainnet.era.zksync.io',
      excludeFromActivityApi: true, // excluding until it's synced
    },
```

Reply:
Good point. Fixed